src/main/java/org/takes/rq/RqForm.java

```diff
@@ -233,4 +233,53 @@ public String single(final CharSequence name, final String def) {
             return value;
         }
     }
+
+    /**
+     * Fake RqForm accepting parameters in the constructor.
+     * @author Matteo Barbieri (barbieri.matteo@gmail.com)
+     */
+    @EqualsAndHashCode(callSuper = true, of = "map")
+    final class Fake extends RqWrap implements RqForm {
+
+        /**
+         * Map of params and values.
+         */
+        private final transient ConcurrentMap<String, List<String>> map;
+
+        /**
+         * Ctor.
+         * @param req Original request
+         * @param params Parameters list
+         */
+        public Fake(final Request req, final String... params) {
+            super(req);
+            if (params.length % 2 != 0) {
+                throw new IllegalArgumentException(
+                    "Wrong number of parameters"
+                );
+            }
+            this.map = new ConcurrentHashMap<String, List<String>>(0);
+            for (int idx = 0; idx < params.length; idx += 2) {
+                final String key = params[idx];
+                final String value = params[idx + 1];
+                if (this.map.containsKey(key)) {
+                    this.map.get(key).add(value);
```

Comment:
@mbarbieri so, if I do:

```java
new RqForm.Fake(
  new RqFake(),
  "foo", "value-1",
  "bar", "value-2",
  "foo", "value-3"
)
```

the `foo` param will have both `value-1` and `value-3`, right?

I don't know, for me it seems "more correct" that the last one overrides the first one... what do you think?